tests/client/service/gestaltsGestaltTrustByNode.test.ts

```diff
@@ -218,6 +218,7 @@ describe('gestaltsGestaltTrustByNode', () => {
       identitiesManager,
       nodeManager,
       sigchain,
+      taskManager,
       logger,
     });
```

Comment:
This is missing `taskManager.startProcessing()`. Maybe processing isn't needed here, I should check it.